resources/views/flights/show.blade.php

```diff
@@ -419,80 +417,95 @@ class="modal fade"
 <script src="https://cdnjs.cloudflare.com/ajax/libs/ajax-bootstrap-select/1.4.5/js/ajax-bootstrap-select.min.js">
 </script>
 
-<script src="https://unpkg.com/leaflet@1.6.0/dist/leaflet.js"
-integrity="sha512-gZwIG9x3wUXg2hdXF6+rVkLF/0Vi9U8D2Ntg4Ga5I5BZpVkVxlJWbSQtXPSiUTtC0TjtGOmxa1AJPuV0CPthew=="
-crossorigin></script>
+<script src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js"
+integrity="sha512-XQoYMqMTK8LvdxXYG3nZ448hOEQiglfqkJs1NOQV44cWnUrBc8PkAOcXy20w0vlaXaVUearIOBhiXZ5V3ynxwA=="
+crossorigin=""></script>
+<script src="https://cdn.jsdelivr.net/npm/leaflet.geodesic"></script>
 
 <script type="text/javascript">
-    $(function () {
+    // Enable tooltips
+    $(function() {
         $('[data-toggle="tooltip"]').tooltip()
     })
 
-    /*
-     * AIRPORT SELECT PICKERS
+    /**
+     * Initialises the live search bootstrap-select pickers
+     *
+     * @param      string     selectPicker  The select picker element (DOM selector)
+     * @param      string     searchRoute   The search route (Laravel route())
+     * @param      boolean    hasSim        Indicates if selectable object has sim property
      */
-    // initialise airport selectpickers with ajax live search
-    $('.selectpicker').selectpicker({
-        liveSearch: true
-    })
-    .ajaxSelectPicker({
-        ajax: {
-            url: '{{ route('search.airport') }}',
-            method: 'GET',
-            data: {
-                q: '@{{{q}}}'
-            }
-        },
-        locale: {
-            emptyTitle: 'Start typing to search...',
-            statusInitialized: '',
-        },
-        preprocessData: function(data){
-            var airports = [];
-            let count;
-            if(data.length > 0){
-                if(data.length >= 10) {
-                    count = 10;
-                } else {
-                    count = data.length;
+    function initLiveSearch(selectPicker, searchRoute, hasSim)
+    {
+        $(selectPicker).selectpicker({
+            liveSearch: true
+        }).ajaxSelectPicker({
+            ajax: {
+                url: searchRoute,
+                method: 'GET',
+                data: {
+                    q: '@{{{q}}}'
                 }
-                for(var i = 0; i < count; i++){
-                    var curr = data[i];
-                    airports.push(
-                        {
-                            'value': curr.icao,
-                            'text': curr.icao + ' - ' + curr.name,
-                            'disabled': false
+            },
+            locale: {
+                emptyTitle: 'Start typing to search...',
+                statusInitialized: '',
+            },
+            preprocessData: function(data)
+            {
+                var aircraft = [];
```

Comment:
Change this to a `let` and also change the name from `aircraft` to something like `result`

Reply:
Resolved in c2d8a63d5e3083ab574bae6541d6b3f3c49f0e19